src/services/ledger/admin.ts

```diff
@@ -97,4 +104,50 @@ export const admin = {
       return new UnknownLedgerError(err)
     }
   },
+
+  addSwapFeeTxSend: async ({
+    swapFeeMetadata,
+    description,
+  }: {
+    swapFeeMetadata: SwapTransactionMetadataUpdate
+    description: string
+  }): Promise<LedgerJournal | LedgerServiceError | DuplicateError> => {
+    const totalSwapFee =
+      swapFeeMetadata.offchainRoutingFee +
+      swapFeeMetadata.onchainMinerFee +
+      swapFeeMetadata.serviceProviderFee
+    try {
+      // check for duplicates
+      const result = await txMetadataRepo.findByHash(swapFeeMetadata.hash)
+      if (result instanceof CouldNotFindTransactionMetadataError) {
+        const bankOwnerWalletId = await getBankOwnerWalletId()
+        const bankOwnerPath = toLiabilitiesWalletId(bankOwnerWalletId)
+        const entry = MainBook.entry(description)
+          .credit(lndAccountingPath, totalSwapFee, {
+            currency: WalletCurrency.Btc,
+            pending: false,
+            type: LedgerTransactionType.Fee,
+          })
+          .debit(bankOwnerPath, totalSwapFee, {
+            currency: WalletCurrency.Btc,
+            pending: false,
+            type: LedgerTransactionType.Fee,
+          })
```

Comment:
This should use the `EntryBuilder`

Reply:
going to refactor this with the Ledger Cleanup work with Arvin in the next PR